scylla-cdc-base/src/main/java/com/scylladb/cdc/cql/CQLConfiguration.java

```diff
@@ -200,7 +216,12 @@ public Builder addContactPoint(String host, int port) {
         }
 
         public Builder addContactPoint(String host) {
-            return addContactPoint(host, DEFAULT_PORT);
+            return addContactPoint(host, defaultPort);
+        }
+
+        public Builder withDefaultPort(int port) {
+            this.defaultPort = port;
+            return this;
```

Comment:
This makes order of adding matter. Let's just add a comment or mark for fixing later.